examples/nirfsa/getting-started-iq.py

```diff
@@ -148,11 +148,14 @@ def raise_if_initialization_error(response):
     error_message = rpc_error.details()
     if rpc_error.code() == grpc.StatusCode.UNAVAILABLE:
         error_message = f"Failed to connect to server on {SERVER_ADDRESS}:{SERVER_PORT}"
+
```

Comment:
You'll find that extra whitespace is not optional with the formatter.

Reply:
Consider using black locally. We have some instructions here: https://github.com/ni/grpc-device/wiki/Development-Tips-and-Tricks#black-format-on-save